### PR TITLE
feat: add scraper for NH, USA and fix CT, USA

### PR DESCRIPTION
**Pull request by qgolsteyn**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
     "no-shadow": "warn",
     "import/extensions": ["error", "ignorePackages"],
     "no-underscore-dangle": "off",
-    "jest/no-try-expect": "off"
+    "jest/no-try-expect": "off",
+    "no-loop-func": "off"
```
Comment (camjc) [inline .eslintrc.json]: Can we please get a rationale of why we’d remove this rule globally and why it’s appropriate in this particular case? https://eslint.org/docs/rules/no-loop-func

File: src/events/crawler/lib/pdf.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+ *
+ * @param {*} target item to find nearest for
+ * @param {*} data array of all PDF items
+ */
+export const getNearest = (target, data) => {
+  return data
+    .filter(item => item.page === target.page) // ignore elements not on the target's page
+    .sort((a, b) => euclideanDistance(target, a) - euclideanDistance(target, b)) // sort by smallest distance
+    .map(item => item.text); // we only want text
+};
```
Comment (camjc) [inline src/events/crawler/lib/pdf.js]: Wow, some hectic PDF hurdles. Nice stuff.

File: src/events/crawler/scrapers/USA/NH/index.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    'Coos',
+    'Grafton',
+    'Carroll',
+    'Belknap',
+    'Merrimack',
+    'Sullivan',
+    'Cheshire',
+    'Hillsborough',
+    'Rockingham',
+    'Strafford'
```
Comment (camjc) [inline src/events/crawler/scrapers/USA/NH/index.js]: NaBD but may as well sort to reduce future churn

**Comment by qgolsteyn** [comment]
@camjc Ready for another round of review!

File: .eslintrc.json (hunk 1 above)
Comment (lazd) [reply .eslintrc.json]: Yeah, where was this biting you @qgolsteyn?

**Comment by lazd** [comment]
I won't have time to review this, but I say if @camjc is good with it, we merge!

File: src/events/crawler/lib/fetch.js [hunk 4]
```diff
@@ -55,15 +55,18 @@ export const fetch = async (url, type, date = process.env.SCRAPE_DATE || datetim
...
-    const fetchedBody = toString ? response.body.toString() : response.body;
-
-    await caching.saveFileToCache(url, type, date, fetchedBody);
 
     if (disableSSL) {
       delete process.env.NODE_TLS_REJECT_UNAUTHORIZED;
     }
 
-    return fetchedBody;
+    if (response.statusCode < 400) {
```
Comment (camjc) [inline src/events/crawler/lib/fetch.js]: or do we just want 200s? (how to handle redirects? will they already be handled before this path?)

**Pull request merged**